src/modules/package.rs

```diff
@@ -175,6 +186,8 @@ fn extract_meson_version(file_contents: &str) -> Option<String> {
 fn get_package_version(base_dir: &Path, config: &PackageConfig) -> Option<String> {
     if let Ok(cargo_toml) = utils::read_file(base_dir.join("Cargo.toml")) {
         extract_cargo_version(&cargo_toml)
+    } else if utils::find_file(base_dir, |file_name| file_name.ends_with(".nimble")).is_some() {
```

Comment:
Can you have this at least make the initial check for `nimble` files via `context`, like the language modules do? It keeps a list of files and file extensions.
```rust
context
  .try_begin_scan()?
  .set_extensions(&["nimble"])
  .is_match();
```